project/Dependencies.scala

```diff
@@ -42,9 +42,7 @@ object Dependencies {
 
     "com.google.guava"               % "guava"               % "19.0",
 
-    excludeGuavaJDK5("com.google.api-client"          % "google-api-client"   % "1.23.0"),
```

Comment:
this lib is not actually removed from the codebase; it's pulled in as a transitive dependency from other google libs, e.g. `google-api-services-cloudbilling`. But, since I removed the code that directly referenced the `google-api-client`, I've removed it as a direct dependency.